Readme.md

```diff
@@ -115,6 +115,9 @@ var nightmare = Nightmare({
 });
 ```
 
+##### ['web-preferences'].preload
+Specifies a file to add to preload.  The generated preload file is written to Electron's `userData` path.
```

Comment:
I agree this line isn't particularly clear, and I struggled with making something concise.

I think "...add to Electron's `preload`" is a bit imprecise as the user's preload is being spliced into the Nightmare preload.  I wanted to give enough information to make it clear that the file provided was not the file used, and where it was saved to on disk without getting into too much detail.  

How about something like:

>Specifies a file to add to Nightmare's preload.  The generated preload file is written to Electron's `userData` path.